meta-lmp-bsp/conf/machine/include/lmp-mfgtool-machine-custom.inc

```diff
@@ -139,6 +139,7 @@ EXTRA_IMAGEDEPENDS:remove:k3 = "u-boot-default-script"
 PREFERRED_PROVIDER_virtual/optee-os:k3 = "optee-os-fio-mfgtool"
 KERNEL_IMAGETYPE:k3 = "fitImage"
 KERNEL_CLASSES:k3 = " kernel-lmp-fitimage "
+UBOOT_SIGN_ENABLE:k3 = "0"
```

Comment:
Why?

Reply:
I think it could be because now the "machine file" includes this variable so this line would be useless anyway

https://git.yoctoproject.org/meta-ti/commit/meta-ti-bsp/conf/machine/include/k3.inc?id=bde91baf8a3b398170388ffb9835979ebfc2e4c4

Reply:
I remove this and postpone to part 4